src/gd_gd.c

```diff
@@ -60,7 +60,7 @@
 #include "gd_errors.h"
 
 /* 2.3: gd is no longer mandatory */
-#if ENABLE_GD_FORMATS
+#ifdef ENABLE_GD_FORMATS
```

Comment:
`#if ENABLE_GD_FORMATS` works fine in my environment. Did you encounter any problems, please? :)

Reply:
The issue is that the resulting library does not have support for the GD formats, unless this change is done.

Reply:
When the option `ENABLE_GD_FORMATS` is set, there will be the following code in `src/config.h`:

```
#define ENABLE_GD_FORMATS 1
```

which make `#if ENABLE_GD_FORMATS` equal to `#if 1` and work fine.